src/Surfnet/StepupMiddleware/ApiBundle/Identity/Entity/RaSecondFactor.php

```diff
@@ -140,6 +151,7 @@ public function jsonSerialize()
             'status'           => (string) $this->status,
             'identity_id'      => $this->identityId,
             'name'             => $this->name,
+            'document_number'  => (string) $this->documentNumber,
```

Comment:
I'd rather have `document_number` be null when it's null (and leave presentation of an absent document number to the RA view layer).